ucf-document.php

```diff
@@ -25,6 +25,7 @@
  * Includes
  */
 require_once 'includes/document-post-type.php';
+require_once 'includes/directory-taxonomy.php';
```

Comment:
We're going to need to call `Directory->register()` in `activate()` as well to ensure term permalinks are flushed on activation